File: Codeforces/581A_Vasya_the_Hipster.py
Red, Blue = map(int, input().split())

if Red < Blue:
    NewRed = Red - Red
    NewBlue = Blue - Red
    print(Red, NewBlue//2)

else:
    NewRed = Red - Blue
    NewBlue = Blue - Blue
    print(Blue, NewRed//2)
    

File: Codeforces/581A_Vasya_the_Hipster.cpp
#include <iostream>
using namespace std;

int main () {
    int Red, Blue, NewRed, NewBlue;
    cin >> Red >> Blue;

    if (Red < Blue) {
        NewRed = Red - Red;
        NewBlue = Blue - Red;
        cout << Red << " " << NewBlue/2 << endl;
    }
    else {
        NewRed = Red - Blue;
        NewBlue = Blue - Blue;
        cout << Blue << " " << NewRed/2 << endl; 
    }
}   


    